less/components/sidebar.less

```diff
@@ -36,6 +36,9 @@ html.no-js .sidebar {
   &.madewithcode-meme,
   &.madewithcode-poster,
   &.maker-party-2015,
+  &.offline-icebreakers {
+    .pageColorizer(@pinkSidebarColor);
+  }
   &.parapara-page {
     .pageColorizer(@pinkSidebarColor);
   }
```

Comment:
we should merge these two blocks since both share the same rule `.pageColorizer(@pinkSidebarColor);` 😉 

```less
&.teaching-materials,
&.web-lit-basics,
&.web-lit-basics-two,
&.protect-your-data,
&.privacy-basics,
&.madewithcode-landing,
&.madewithcode-firstwebpage,
&.madewithcode-meme,
&.madewithcode-poster,
&.maker-party-2015,
&.parapara-page,
&.offline-icebreakers {
  .pageColorizer(@pinkSidebarColor);
}
```